src/Utils/StyleConfig.test.ts

```diff
@@ -0,0 +1,29 @@
+import { ScreenSize, Device } from './StyleConfig';
+
+describe('ScreenSize', () => {
+  test('ScreenSize for various devices', () => {
+    expect(ScreenSize).toEqual({
```

Comment:
we can do `expect(ScreenSize).toMatchSnapshot()`, but currently our jest setting will run test against compiled js file, so CI will fail, I will update jest setting. 

This is ok for now.